tests/data/tilde_req-expected.json

```diff
@@ -2,7 +2,7 @@
   "headers": {
     "tool_name": "python-inspector",
     "tool_homepageurl": "https://github.com/nexB/python-inspector",
-    "tool_version": "0.9.3",
```

Comment:
Maybe we should remove header information from test files with a function like we do in scancode-toolkit, this will avoid unnecessary churn every time we update. Also gets rid of file paths when `--requirement` option is specified. Don't have to do this here of course, maybe in a later version.